apps/nextjs/src/app/layout.tsx

```diff
@@ -143,7 +143,7 @@ export default async function RootLayout({
               suppressHydrationWarning
               // eslint-disable-next-line react/jsx-no-comment-textnodes
             >
-              /* css comment for goober */
+              {/* css comment for goober */}
```

Comment:
I think that this needs to be a genuine css comment, not a comment that gets stripped out when we build. This is likely causing the e2e failure and reflects the Sentry error mentioning "goober"